src/visualization/GraphRenderer.h

```diff
@@ -97,7 +102,9 @@ class GraphRenderer{
          * \param event a Event which will describe the event that will happen
          * \param nodeList a vector of Nodes which is needed for a few methods
          */
-        void eventHandler(sf::Event event, sf::RenderWindow& window, vector<Node>& nodeList);
+        void eventHandler(sf::Event event, sf::RenderWindow& window, vector<Node>& nodeList, vector<DrawNode>& Nodes, state& gameState);
+        void scoreHandler(sf::Event event, sf::RenderWindow& window);
+        void updateDrawNode(sf::RenderWindow& window, vector<Node>& nodeList, vector<DrawNode>& Nodes);
```

Comment:
Not sure what the purpose of this method is. It this the interpolation of the node positions when layout has changed?
In any case: The `GraphRenderer` should not write node positions, only read. For any changes made to the `DrawNode`s, better make a separate class.

Reply:
I'll do soon. I just used the GraphRenderer temporarly to realize the idea for now.